pkg/adaptor/hypervisor/registry/aws.go

```diff
@@ -1,13 +1,13 @@
+//go:build aws
 // +build aws
```

Comment:
nit: while here maybe remove the old style tag

Reply:
Fixed